packages/trader/src/AppV2/Components/TradeParameters/Barrier/barrier.tsx

```diff
@@ -52,7 +56,13 @@ const Barrier = observer(({ is_minimized }: TDurationProps) => {
                 onClick={() => setIsOpen(true)}
                 className={clsx('trade-params__option', is_minimized && 'trade-params__option--minimized')}
             />
-            <ActionSheet.Root isOpen={is_open} onClose={() => onClose(false)} position='left' expandable={false}>
+            <ActionSheet.Root
+                isOpen={is_open}
+                onClose={onClose}
+                position='left'
+                expandable={false}
+                shouldBlurOnClose={is_open}
```

Comment:
does it need a variable? that setting should be fairly static, right?

Reply:
Unfortunately it temporary needs a variable, because it turns out, that close function is called not only closing (because close func  was added to array with dependencies:( ).
I'm going to remove this functionality as it's better not to use TextField component for our purpose. The reason is that TextField is based on input tag (for which having focus is a normal state), but we don't allow user to type in it, only click. So the idea is to create another component for our needs, something like label, without input, and revert changes with bluring